week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -94,3 +171,196 @@ function testAll() {
 }
 
 testAll();
+
+/*https://developers.google.com/web/tools/chrome-devtools/console/*/
+
+
+
+
+/* the switch statement explained  - https://www.w3schools.com/js/js_switch.asp   The switch expression is evaluated once, then the value of the expression is compared with the values of each case. If there is a match, the associated block of code is executed. When JavaScript reaches a break keyword, it breaks out of the switch block. This will stop the execution of inside the block. It is not necessary to break the last case in a switch block. The block breaks (ends) there anyway.
+Switch cases use strict comparison (===). The values must be of the same type to match. A strict comparison can only be true if the operands are of the same type.*/
+
+// magicBall version 2 - used a switch statement and an if else if statement for the magicBall excerzise to understand switch more - two parts to a switch statement -
+//part one - function one - indexes all the possibly answers randomly
+//source - https://discuss.codecademy.com/t/help-with-javascript-8-ball-project/68618/7
+
+function shakeBall() {
```

Comment:
Which of these 2 implementations (a fancy word for a way of writing some code!) do you think is the best? Why do you think that?